bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -242,15 +242,15 @@ spec:
               - key: truststore
                 path: truststore
         {{- end }}
-        {{- if or (.Files.Glob "files/conf/*") .Values.existingConfiguration }}
+        {{- if .Values.existingConfiguration }}
         - name: configurations
           configMap:
-            name: {{ include "cassandra.configurationCM" . }}
+            name: {{ .Values.existingConfiguration }}
```

Comment:
I see that you removed the helper function. This causes that umbrella charts cannot redefine the helper functions (example: Gitlab chart). If you want to remove the helper it's fine but then allow existingConfiguration and initDBConfigMap to templatized